std/typecons.d

```diff
@@ -2999,24 +3000,32 @@ Returns:
     assert(i == 5);
 }
 
+    deprecated("Deprecated to rely on implicit Nullable.get. Please use .get explicitly instead.")
```

Comment:
Grammar. Maybe "Implicit conversion through Nullable's alias this is deprecated. Please use .get explicitly instead."?

Ditto throughout.

Reply:
better?

Reply:
Yes, thanks!